classes/helper.php

```diff
@@ -340,7 +348,7 @@ public static function plain_text_search(string $search, string $table,
      */
     public static function regular_expression_search(string $search, string $table,
                                                      database_column_info $column, bool $summary = false,
-                                                     $stream = null): array {
+                                                     $stream = null, $linkfunction = null): array {
```

Comment:
I don't like how this is passed in as a function, it should just use it as its own internal dependancy